app/views/user_mailer/account_verified.html.erb

```diff
@@ -1,15 +1,67 @@
-<p class="lead">
+<%= image_tag(
+      attachments['verified.png'].url,
+      width: 110,
+      height: 80,
+      alt: '',
+      role: 'img',
+      class: 'float-center padding-bottom-4',
+    ) %>
+
+<h1><%= message.subject %></h1>
+
+<p>
+  <%= t('user_mailer.account_verified.greeting') %>
+</p>
+<p>
+  <%= t('user_mailer.account_verified.intro', date: @date) %>
+</p>
+
+<p>
+<% if @presenter.service_provider.present? %>
```

Comment:
Our presenter draws on V`erificationResultsEmailPresenter` for inspiration. The logic here is probably best captured in tests.